genai-on-vertex-ai/natural_language_to_sql/natural_language_to_sql.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    Given the following movie gener trends data:
</pre><p>Typo: genre</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>